### PR TITLE
hfss fixes for export_touchstone()

### PR DESCRIPTION
**Pull request by gmalinve**
In this PR export_touchstone() has been fixed for hfss, hfss3D and Circuit in order to make solution_name, sweep_name and file_name as default values and fix for variations.

### CODE REVIEW

**Review by maxcapodi78 — APPROVED** [review]
LGTM

File: pyaedt/application/Analysis.py [hunk 1]
```diff
@@ -1709,3 +1709,122 @@ def submit_job(
...
+            ext = ".S" + n + "p"
+            filename = os.path.join(self.working_directory, solution_name + "_" + sweep_name + appendix + ext)
+        else:
+            filename = file_name.replace("//", "/").replace("\\", "/")
+        self.logger.info("Exporting Touchstone " + filename)
+        DesignVariations = ""
+        i = 0
+        for el in variations:
+            DesignVariations += str(variations[i]) + "='" + str(variations_value[i].replace("'", "")) + "' "
+            i += 1
```
Comment (maxcapodi78) [inline pyaedt/application/Analysis.py:1776]:
```suggestion
        for i in range(len(variations)):
            DesignVariations += str(variations[i]) + "='" + str(variations_value[i].replace("'", "")) + "' "
```

**Review by maxcapodi78 — APPROVED** [review]
Looks good apart a small enhancement to fix Codacy error and improve code quality

**Review by Alberto-DM — COMMENTED** [review]
Tested successfully.

**Pull request merged**